region/crateria/east.json

```diff
@@ -80,34 +108,58 @@
           "to": [
             {
               "id": 1,
-              "requires": null
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": null
+                }
+              ]
             },
             {
               "id": 2,
-              "requires": [
-                {"or": [
-                  "SpaceJump",
-                  "canCWJ",
-                  "Grapple",
-                  "canSpringBallJump",
-                  "canHorizontalBombJump",
-                  {"canComeInCharged": {
-                    "fromNode": 1,
-                    "framesRemaining": 0,
-                    "shinesparkFrames": 50
-                  }},
-                  {"and": [
-                    "Gravity",
-                    {"or": [
-                      "HiJump",
-                      "canGravityJump",
-                      "canSpringBallJumpMidAir",
-                      "canIBJ"
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": [
+                    {"or":[
+                      "SpaceJump",
+                      "Grapple",
+                      {"canComeInCharged": {
+                        "fromNode": 1,
+                        "framesRemaining": 0,
+                        "shinesparkFrames": 50
+                      }},
+                      {"and": [
+                        "Gravity",
```

Comment:
This should be split into an `AquamanRises` series of strats.
gravity jump, springball jump, and ibj are all notable tricks imo

Reply:
What I've tried to do with notable is that if there's nothing unusual about using that tech vs in other places, I don't make it notable. That is, if I figure being able to execute the tech should allow you to execute the strat without further practice, I won't make it notable, except if the specific trick is quite well-known (e.g. PCJR short charge).

The IBJs I've made notable have usually been longer ones, or ones starting in midair (which I want to make a tech for sometime, and a helper allowing you to substitute SpeingBall + regular IBJ). Gravity Jump and vertical MidAir Springball are likewise usually very standard (but for example, their use is notable in Pants Room). Springwalls are pretty much all notable because many of them are different.

I've not done that for X-Ray climb (I made them all notable) and questioned that for a while, but now I feel like those are cases where it's much better to know exactly where the climb ends which you can't really tell by winging it.

I've thought about making a helper for jump assist things, but I'm not sure it helps clarity because what works or doesn't is pretty variable from one place to the next (sometimes hj works and SpringBall doesn't. Sometimes you need both). Maybe `hasSingleJumpAssist` and `hasTwoJumpAssists` could hold value, but in the end it's just for readability.

A thing I'm noticing is as time went on, I started splitting things into more strats compared to earlier in the refactor.